extensions-compose/src/commonMain/kotlin/com/arkivanov/decompose/extensions/compose/stack/animation/predictiveback/PredictiveBackAnimation.kt

```diff
@@ -48,7 +48,7 @@ fun <C : Any, T : Any> predictiveBackAnimation(
 ): StackAnimation<C, T> =
     PredictiveBackAnimation(
         backHandler = backHandler,
-        animation = fallbackAnimation ?: emptyStackAnimation(),
+        animation = fallbackAnimation ?: LocalStackAnimation.current,
```

Comment:
I believe this won't compile because type parameters (generics) do not match?

Reply:
I think this could be done as follows, but let me first think wether it's worth it or not. I will update in the issue.

```kotlin
val LocalStackAnimation =
    compositionLocalOf<DefaultStackAnimationProvider> {
        object : DefaultStackAnimationProvider {
            override fun <C : Any, T : Any> invoke(): StackAnimation<C, T> =
                emptyStackAnimation()
        }
    }

interface DefaultStackAnimationProvider {
    operator fun <C : Any, T : Any> invoke(): StackAnimation<C, T>
}
```

Reply:
I've fixed it and tested it - apologies for the initial broken PR